appsDiscovery/src/main/java/org/kinecosystem/appsdiscovery/view/customView/AppsDiscoveryButton.kt

```diff
@@ -11,16 +11,36 @@ import org.kinecosystem.appsdiscovery.view.AppsDiscoveryActivity
 class AppsDiscoveryButton @JvmOverloads constructor(context: Context, attrs: AttributeSet? = null, defStyleAttr: Int = 0) :
         TextView(context, attrs, defStyleAttr) {
 
-    private fun startAppsDiscoveryActivity() {
-        context.startActivity(AppsDiscoveryActivity.getIntent(context))
+    private fun startAppsDiscoveryActivity(debugMode: Boolean) {
```

Comment:
probably this is wrong to be private 
need to change to public and debug should have default false

Reply:
method is actually not necessary